scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -393,6 +393,15 @@ boolean preemptTask(JobId jobId, String taskName, int restartDelay)
      */
     boolean removeJobs(List<JobId> jobIds) throws NotConnectedException, PermissionException;
 
+    /**
+     * Remove jobs older than the given epoch time (jobs which were finished earlier)
+     * @param olderThan epoch time to consider
+     * @return true if all jobs with jobIds were removed, otherwise false
+     * @throws NotConnectedException if you are not authenticated
+     * @throws PermissionException if you can't access to at least one of the job
```

Comment:
@throws NotConnectedException if **the user is** not authenticated
@throws PermissionException if **the user**  can't access to at least one of the job**s**

Reply:
because it is written like this in all other methods (and I am not going to change them all today)